pkg/crdconversion/crdconversion.go

```diff
@@ -195,6 +195,15 @@ func updateCrdConversionWebhookConfiguration(cert certificate.Certificater, crdC
 		},
 	}
 
+	if enableReconciler {
+		existingLabels := crd.Labels
+		if existingLabels == nil {
+			existingLabels = map[string]string{}
+		}
+		existingLabels[constants.ReconcileLabel] = "yes"
```

Comment:
```suggestion
		existingLabels[constants.ReconcileLabel] = "true"
```

Just so we can convert string to bool easily using `strconv.ParseBool`.

Reply:
Ack, I will split the PR